variants/github_actions_ci/workflows/ci.yml.tt

```diff
@@ -95,6 +95,13 @@ jobs:
       - run: bundle exec bundle audit --update
       - run: bundle exec rails db:setup
       - run: bundle exec rspec spec --format progress
+      - name: Archive spec outputs
+        uses: actions/upload-artifact@v3
+        if: success() || failure()
```

Comment:
This means it will run even if RSpec fails (we _especially_ want it to run when RSpec fails). I've set this to still run on success, because some test suits do retries, so maybe there are still interesting screenshots.